### PR TITLE
Multiple Reactions UI fixes

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

This PR fixes two things from #19064 and generates screenshots for one more task which was finished in #19034

- [x] Reaction Counter (Number) styling looks different for Reactions vs Comment and save. The number under Reactions is bolded
Image

- [ ] Remove the circle highlight styling for Reaction and Save. Our icon + color is enough to convey the active state. Please add screenshots for desktop/mobile when addressing and tag @anujbhavsar96 on the PR for a second look.
Image

- [ ] Timing issues on front-end when multiple reactions happen quickly: While registering multiple reactions quickly, not all reactions flash on the **_left action toolbar_** (see where Heart+ icon is ---> it should update to last clicked on icon). We could remove this animation where we are flashing the Reaction icon on the left action toolbar to display the system response since we already have ringpop animation.
Image

## Related Tickets & Documents

- Related Issue #19064

## QA Instructions, Screenshots, Recordings

**How to test?**
1. Enable the `multiple_reactions` FeatureFlag at `/admin/feature_flags/features`
2. View an article
3. Play with the reactions and bookmark icon.

### The UI for counter text is same in all cases and looks correct & the circle for highlight icon has been removed too.

Screenshot 2023-02-07 at 7 04 19 PM Screenshot 2023-02-07 at 7 04 36 PM

### The reaction-drawer-button animation has been removed as a result the counter and internal icon animation does work perfectly.
https://user-images.githubusercontent.com/9396084/217258985-cf87f1a9-d7e7-40ba-848f-f316c0477884.mov

### UI accessibility concerns?

No

## Added/updated tests?

- [x] No

## [Forem core team only] How will this change be communicated?
- [x] This change does not need to be communicated

### CODE REVIEW

**Comment by rt4914** [comment]
@anujbhavsar96 Can you check the screenshots/recordings and confirm that it matches your expectation.

**Comment by anujbhavsar96** [comment]
@rt4914 

> The UI for counter text is same in all cases and looks correct & the circle for highlight icon has been removed too.

This looks good. But can we also remove the color from the bookmark count when it's active for consistency? We can keep the same gray color as the reaction and comment.
Feb-08-2023 09-39-11

> The reaction-drawer-button animation has been removed as a result the counter and internal icon animation does work perfectly.

Looks much better without flashing animation, but when you removed all your reactions, the heart+ icon should be changed to the first state (non-filled gray icon). Check video at 0:07.

**Comment by rt4914** [comment]
> @rt4914
> 
> > The UI for counter text is same in all cases and looks correct & the circle for highlight icon has been removed too.
> 
> This looks good. But can we also remove the color from the bookmark count when it's active for consistency? We can keep the same gray color as the reaction and comment. SCR-20230208-ffa
> 
> > The reaction-drawer-button animation has been removed as a result the counter and internal icon animation does work perfectly.
> 
> Looks much better without flashing animation, but when you removed all your reactions, the heart+ icon should be changed to the first state (non-filled gray icon). Check video at 0:07.

@anujbhavsar96 Removed blue color for bookmark
Screenshot 2023-02-08 at 7 20 21 PM Screenshot 2023-02-08 at 7 22 44 PM

**Review by anujbhavsar96 — APPROVED** [review]

File: app/assets/javascripts/initializers/initializeArticleReactions.js [hunk 1]
```diff
@@ -46,28 +46,6 @@ function showUserReaction(reactionName, animatedClass) {
...
-      activeDrawerIcon.outerHTML = activeIcon.outerHTML;
-
-      setTimeout(function () {
-        document
-          .getElementById('reaction-drawer-trigger')
-          .querySelector('.crayons-reaction__icon--active img').outerHTML =
-          reactionDrawerButton.originalIcon;
-      }, 1500);
-    }
-  }
```
Comment (jaw6) [inline app/assets/javascripts/initializers/initializeArticleReactions.js:70]: 😢 to see it go - it was a fun little animation - but better to be plain than to be buggy, and I believe we'll get it sorted out eventually.

**Review by jaw6 — APPROVED** [review]

**Pull request merged**